tests/api/test_energy_robotics_api.py

```diff
@@ -40,6 +48,59 @@ def test_fails_if_status_is_anything_else(self):
             api.pause_current_mission("test_exr_robot_id")
 
 
+@mock.patch(
+    "isar_exr.api.graphql_client.get_access_token",
+    mock.Mock(return_value="test_token"),
+)
+class Test_create_point_of_intrest:
```

Comment:
Following the other test classes the class name should be in UpperCamelCase: `TestCreatePointOfInterest`